tree/20_cfe_basics/ncf_lib.cf

```diff
@@ -1014,10 +1048,12 @@ bundle agent ncf_services(service, action)
       "method"                        string => "lsitab/chitab";
 
     # /etc/rcX.d/
-    pass1.is_boot_action.is_check_action.(!systemctl_utility_present|is_init_service).!is_upstart_service.!svcadm_utility_present.!chkconfig_utility_present.!chitab_utility_present::
+    pass1.is_boot_action.is_check_action.(!systemctl_utility_present|is_init_service).!is_upstart_service.!svcadm_utility_present.!chkconfig_utility_present.!chitab_utility_present.!slackware::
       "action_command" string => "${paths.path[test]} -f /etc/rc`/sbin/runlevel | ${paths.path[cut]} -d' ' -f2`.d/S??${service}";
       "method"         string => "/etc/rcX.d/";
 
+    # Slackware boot actions are not performed by command, they are implemented in the 'methods' promises
```

Comment:
It is left here because in the previous "non-boot actions" block, we are defining action_commands and/or methods for the other platforms, so one may expect to find Slackware boot actions at this place. As we do not perform those actions in "vars" promises, this is explaining where to find them